src/Resources/app/administration/src/module/sw-extension/component/sw-extension-uninstall-modal/sw-extension-uninstall-modal.html.twig

```diff
@@ -0,0 +1,8 @@
+{% block sw_installed_extension_card_removal_content_modal_body %}
+    <div
+        v-if="['Language pack', 'Sprachpaket'].includes(extensionName)"
```

Comment:
Not ideal, but apparently there's no further context information about the current extension.